crates/uv-python/src/version_files.rs

```diff
@@ -70,6 +70,7 @@ impl PythonVersionFile {
                         let trimmed = line.trim();
                         !(trimmed.is_empty() || trimmed.starts_with('#'))
                     })
+                    .flat_map(|line| line.split_whitespace())
```

Comment:
`pyenv` allows providing multiple versions in one line. 
especially when one uses `pyenv local list_of_python_versions` all will be put in one line.